db/migrate/20160911130449_add_value_serialized_to_custom_attribute.rb

```diff
@@ -0,0 +1,19 @@
+class AddValueSerializedToCustomAttribute < ActiveRecord::Migration[5.0]
+  def up
+    add_column :custom_attributes, :serialized_value, :text
+
+    say_with_time "Copying values to serialized_value" do
+      CustomAttribute.all.each do |custom_attribute|
+        custom_attribute.update!(:serialized_value => custom_attribute.value)
+      end
+    end
+  end
+
+  def down
+    remove_column :custom_attributes, :serialized_value
+  end
+
+  class CustomAttribute < ActiveRecord::Base
+    serialize :serialized_value
+  end
```

Comment:
Minor, but since other changes are needed, we put this first right after migration class definition.